.github/workflows/release_ruby.yml

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Release Ruby Binding
+
+on:
```

Comment:
Other bindings use:

```
on:
  push:
    tags:
      - "*"
  pull_request:
    branches:
      - main
    paths:
      - ".github/workflows/release_python.yml"
  workflow_dispatch:
```

I don't see I would publish in PR or tags events.

Reply:
I don't think we need to tag in the release workflow, and in fact, the tag should trigger this release workflow to run.

Our release workflow will be as follows:

- Initiate the release discussion
- Create a release candidate tag such as `v0.54.0-rc.1` for voting
- Create the formal release tag like `v0.54.0` after approval

For every bindings release workflow, the process must handle:

- Verifying the release procedure during the RC phase
- Publishing the release artifacts after the final tag is created

Is this align our current understanding?